vitrina/users/migrations/0001_initial.py

```diff
@@ -82,6 +82,7 @@ class Migration(migrations.Migration):
                 ('year_of_birth', models.IntegerField(blank=True, null=True)),
                 ('disabled', models.BooleanField()),
                 ('suspended', models.BooleanField()),
+                ('organization_id', models.BigIntegerField(blank=True, null=True)),
```

Comment:
Something is messed up with `User` model migrations. `User` model has `organization` property, but it is not added via migrations, When I tried to add it, I get circular dependencies with `orgs` migrations, because both depend on one another.

So I tried to add just a `BitIntergerField` here and later  in `vitrina/users/migrations/0003_auto_20220816_1153.py` change it into `ForeignKey`, but now migrations work, but I get:

```python

Traceback (most recent call last):
  File "django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
The above exception (column user.organization_id does not exist
LINE 1: ...r"."last_name", "user"."password", "user"."role", "user"."or...
                                                             ^
HINT:  Perhaps you meant to reference the column "user.organization_id_id".
) was the direct cause of the following exception:
```

So I guess, first wee need to add just `organization` and then alter it to foreign key.

Could you pleas check it out and fix?